pkg/stringmapprovider/stringmapprovider.go

```diff
@@ -21,6 +23,8 @@ func New(provider api.StaticConfig) (api.LazyLoadedStringMapProvider, error) {
 		return awssec.New(provider), nil
 	case "sops":
 		return sops.New(provider), nil
+	case "secretmanager":
```

Comment:
Can we use `gcpsecrets` for this one?

Context: I've used `awssecrets` rather than `secretsmanager` for AWS Secrets Manager, hoping this will allows us to distinguish which secrets manager we're referring.

Reply:
👍